plugins/generator-1.19.4/forge-1.19.4/mappings/itemproperties.yaml

```diff
@@ -5,4 +5,5 @@ damage: damage
 lefthanded: lefthanded
 cooldown: cooldown
 custom_model_data: custom_model_data
+blocking: blocking

```

Comment:
What happens if a user uses this property in item ME where the blocking property does not exist?

Reply:
Hmm, should it be a reserved name in item editor then?

Reply:
Is it even needed on the list? I would just remove it from the list

Reply:
In that case it would have mod ID inserted as the namespace, which breaks reference to the vanilla property.

Reply:
You could use datalists "type" property to determine that. Eg. if it is not item type, not show it in items ME, similar to eg. gamerules or blocksitems

Reply:
The issue is the `type` field is already used for the property's value type (Logic, Integer, Number, Text). Use the `other` field?

Reply:
>What happens if a user uses this property in item ME where the blocking property does not exist?

Oh, wait, I totally forgot, in item editor it would have `CUSTOM:` prefix as a property created by an end user, in tool editor it is created programmatically and doesn't have that prefix, and names of properties without it are not modified when calling `PropertyData#getPrefixedName(String)`.

Reply:
Ok so technically this property could be skipped from mappings?

Reply:
Yeah, I think so.